src/main/java/Serializer.java

```diff
@@ -0,0 +1,40 @@
+import java.util.regex.Pattern;
+
+public class Serializer {
+    public String SerializeList(List list){
```

Comment:
i would suggest, that you define a common standard how to save it, and put the same in file, and database, so you can easily extend it later.